README.md

```diff
@@ -240,6 +240,51 @@ var calendarInstance = $calEl.data('tuiCalendar');
 calendarInstance.createSchedules([...]);
 ```
 
+### Custom Timezone
+There are three ways to specify a primary time zone (eg'Asia / Seoul','America / New_York') when rendering a calendar.
+
+1\) If a `timezoneOffset` is specified, it is calculated as a fixed offset value regardless of summer time.
+
+```js
+var cal = new Calendar('#calendar', {
+  timezones: [
+    {
+      timezoneOffset: 540,
+      tooltip: 'Seoul',
+    },
+  ]
+});
+```
+
+2\) If your browser supports `Intl.DateTimeFormat` (including `formatToPart`), you can calculate by entering only the `timezone` property.
+
+```js
+var cal = new Calendar('#calendar', {
+  timezones: [
+    {
+      tooltip: 'New York',
+      timezone: 'America/New_York',
```

Comment:
The priority of `timezoneOffset` is higher than that of `timezone`.
In the case of a time zone where summer time is not applied, it can be calculated only with an offset value.

What is your opinion?